src/engine/framework/VirtualMachine.cpp

```diff
@@ -235,7 +235,7 @@ static std::pair<Sys::OSHandle, IPC::Socket> CreateNaClVM(std::pair<IPC::Socket,
 	CheckMinAddressSysctlTooLarge();
 	const std::string& libPath = FS::GetLibPath();
 #ifdef NACL_RUNTIME_PATH
```

Comment:
That's not right since the new definition does not use a macro.

Reply:
Fixed.